Misc/NEWS.d/next/Library/2024-02-08-17-04-58.gh-issue-112903.SN_vUs.rst

```diff
@@ -0,0 +1,2 @@
+Fix "issubclass() arg 1 must be a class" errors in certain cases of multiple
+inheritance with generic aliases.
```

Comment:
This PR fixes older and more fundamental bug that existed before 3.13. BTW, 3.13 has not yet been released, so we cannot talk about regression in 3.13.

Reply:
3.13 alphas have already been released. The bug is already impacting early testers of Python 3.13, or it would never have been reported in the first place. Early testers of Python 3.13 will surely find it useful to know that this bug has been resolved.

I agree that there was a latent bug prior to 3.13, but it wasn't visible to users, so I'm not sure it should be mentioned in the news entry.

Perhaps something like this?

```suggestion
Fix regression in early Python 3.13 alphas where certain cases of multiple
inheritance with generic aliases would cause spurious "issubclass() arg 1 must
be a class" errors.
```

Reply:
I added a mention that this was a regression in early 3.13 alphas.